packages/itinerary-body/src/defaults/route-description.js

```diff
@@ -0,0 +1,34 @@
+import { legType } from "@opentripplanner/core-utils/lib/types";
+import React from "react";
+
+import * as Styled from "../styled";
+
+export default function RouteDescription({ leg }) {
+  const { headsign, routeLongName, routeShortName } = leg;
+  return (
+    <Styled.LegDescriptionForTransit>
+      {routeShortName && (
+        <div>
+          <Styled.LegDescriptionRouteShortName>
+            {routeShortName}
+          </Styled.LegDescriptionRouteShortName>
+        </div>
+      )}
+      <Styled.LegDescriptionRouteLongName>
+        {routeLongName}
+        {headsign && (
+          <span>
+            <Styled.LegDescriptionHeadsignPrefix>
+              {" to "}
```

Comment:
I think you can remove `{"`

Reply:
The spaces are needed, otherwise, the words will get squished together.